rgbd/test/image_gtest.cpp

```diff
@@ -0,0 +1,79 @@
+#include "test_utils.h"
+
+#include <gtest/gtest.h>
+
+#include <opencv2/core/mat.hpp>
+
+#include <rgbd/image.h>
+
+#include <sensor_msgs/CameraInfo.h>
+
+
+TEST(Image, EmptyClone)
+{
+    rgbd::Image image1;
+    rgbd::Image image2 = image1.clone();
+
+    EXPECT_EQ(image1, image2);
+    EXPECT_FALSE(image1 != image2);
+}
+
+TEST(Image, RandomClone)
+{
+    rgbd::Image image1 = rgbd::generateRandomImage();
+    rgbd::Image image2 = image1.clone();
+
+    EXPECT_EQ(image1, image2);
+    EXPECT_FALSE(image1 != image2);
+}
+
+TEST(Image, NotEqual)
+{
+    rgbd::Image image1 = rgbd::generateRandomImage();
+    rgbd::Image image2 = rgbd::generateRandomImage();
```

Comment:
What are the odds of getting the same random image? I think in practice it would never happen but it would be a hell of a bug to find.

Reply:
Very low. As the color-, depth-image, the timestamp and the frame_id is random.